src/wasm-emscripten.h

```diff
@@ -23,6 +23,8 @@
 
 namespace wasm {
 
+Global* getStackPointerGlobal(Module& wasm);
```

Comment:
this could go in `src/abi/stack` perhaps?

Reply:
I didn't know about `src/abi/stack`,.. it does look like it duplicates some this logic.